### DISCUSSION TITLE
Home screen widget

### DISCUSSION THREAD

**Discussion by MechanicalNoob05** [emavgl/oinkoin#172] [Ideas]
I like the simple UI design very much and have recently started using it and I know this is an open source project and this is not anyone's full time job but would really appreciate a home screen widget for quick expenses adding nothing big just a small small input with four of top expenses or user picked expenses to show in the widget the user can input amount and press any one of four icons to save the it to its particular expense

**Comment by emavgl** [comment]
It would be awesome! Thanks for sharing your idea. My idea would be 2 widget: one with the box that you see at the homepage, with a summary of the income/expense/balance. The second one could be, as you mention, a Widget containing the most used category, when you click in one, it enters directly in the edit page.

Hopefully will get some time to work on this.

Thanks for sharing your ideas!
